personality/xrootd/env.pan

```diff
@@ -35,3 +38,19 @@ variable XROOTD_MONITORING_DESTINATIONS ?= undef;
 variable XROOTD_REPORTING_DESTINATIONS ?= undef;
 
 
+# Default GSI-related parameters
+variable XROOTD_SECURITY_GSI_PARAMS ?= {
+  SELF["ca"] = 2;
+  SELF["cert"] = "/etc/grid-security/dpmmgr/dpmcert.pem";
+  SELF["crl"] = 3;
+  SELF["gmapopt"] = 10;
+  SELF["key"] = "/etc/grid-security/dpmmgr/dpmkey.pem";
+  SELF["md"] = "sha256:sha1";
+  if ( VOMS_XROOTD_EXTENSION_ENABLED ) { 
+    SELF["vomsfun"] = "/usr/lib64/libXrdSecgsiVOMS.so";
+  } else {
+    SELF["vomsat"] = 0;
+  };
+
+  SELF;
+};
```

Comment:
@jrha I looked at your proposal more closely and I don't buy it. I agree that all these `SELF` are not as readable as your version but the 2 last lines with the `if.. else...` block will be the source of future problems. We look the `key/value` structure (at least it is not easy to identify) and at some point there is the risk of somebody changing one line without changing the other. I think my version is more maintainable.... Is it acceptable?